index.js

```diff
@@ -0,0 +1,37 @@
+
+'use strict';
+
+const EE = require('events');
+const ee = new EE();
+const fs = require('fs');
+const objConstructor = require(`${__dirname}/models/bitmap-object.js`).ImageObj;
+const invert = require(`${__dirname}/lib/bitmap-transform.js`);
+const greenScale = require(`${__dirname}/lib/bitmap-greenscale.js`);
+const grayScale = require(`${__dirname}/lib/bitmap-grayscale.js`);
```

Comment:
@khalidM3 @reganoneill @dkulp23 organization of required modules looks good - that said, no need for `${__dirname}` here - you can still use simple relative paths in your `require` statements